### PR TITLE
Added timer option in step

### PR DESCRIPTION
**Pull request by cattyngmd**
works well on const and 2b2tpvp

### CODE REVIEW

**Comment by cattyngmd** [comment]
damn

**Comment by 3arthqu4ke** [comment]
I always ignore the Complex method stuff 😅

File: src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java [hunk 1]
```diff
@@ -103,6 +107,7 @@ protected boolean canStep()
     }
 
     protected void reset() {
```
Comment (3arthqu4ke) [inline src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java:109]: Could set usingTimer to false here too?

**Review by 3arthqu4ke — APPROVED** [review]
Comment doesnt block, can you merge yourself after I approve?

File: src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java (hunk 1 above)
Comment (cattyngmd) [reply src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java:109]: oh yeahhh

Comment (cattyngmd) [reply src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java:109]: wait no, i removed that variable

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java:109]: Ah I meant in Noswlodown

Comment (cattyngmd) [reply src/main/java/me/earth/earthhack/impl/modules/movement/step/Step.java:109]: ah yeah ok

**Comment by 3arthqu4ke** [comment]
AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH

**Comment by 3arthqu4ke** [comment]
@cattyngmd shouldnt we reset the timer on NoSlowDown disable?

**Comment by cattyngmd** [comment]
> @cattyngmd shouldnt we reset the timer (edit Managers.TIMER) on NoSlowDown disable?

hm i think ye, damnn

**Pull request merged**